src/views/Auth/Login.jsx

```diff
@@ -26,23 +26,31 @@ export default function Login() {
     // from React Router to replace the current URL with the URL
     // we need to redirect to.
     // See https://v5.reactrouter.com/web/api/history for the appropriate method to use
+
```

Comment:
And here nice use for this syntax makes your code look clean while it checks for a user to makes sure login was a success